decidim-admin/app/views/layouts/decidim/admin/_sidebar_menu.html.erb

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
There are still some references to settings here

Reply:
Removed here: [52bceae](https://github.com/decidim/decidim/pull/11654/commits/52bceae6951727677dd0f1e9535bea45e7bcf545)

Reply:
Ah ok, I didn't get the chance to try it out on mobile. We should probably add an accordion, but being honesty, we can leave it as is and just fix the whole mobile experience (for instance whats up with the breadcrumb?)

![image](https://github.com/decidim/decidim/assets/717367/9aca4893-b36e-48c3-89e8-4454d567e8c6)


But please leave this problem (mobile experience) out of the current PR!